ccvm_simulators/tests/unit/postprocessor/test_grad_descent.py

```diff
@@ -0,0 +1,99 @@
+import logging
+import torch
+from unittest import TestCase
+from ccvm_simulators.post_processor.grad_descent import PostProcessorGradDescent
+
+
+class TestPostProcessorGradDescent(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.logger = logging.getLogger()
+        cls.default_step_size = 0.1
+        cls.post_processor = PostProcessorGradDescent()
+        cls.N = 20
+        cls.M = 100
+        cls.c = torch.FloatTensor(cls.M, cls.N)
+        cls.q_matrix = torch.FloatTensor(cls.N, cls.N)
+        cls.v_vector = torch.FloatTensor(cls.N)
```

Comment:
Estella, I'm not sure if what we decided about using `torch.FloatTensor` but I think they were producing unpredictable values that were very large or very small. If these values are used in the `test_postprocess_custom_upper_lower_clamp` or the other ones it can generate `nan` or `inf` values and fail the tests. If these (`cls.c`, `cls.q_matrix`, and `cls.v_vector`) are not changed anywhere else, I suggest using the following
```
cls.c = torch.zeros(cls.M, cls.N)
cls.q_matrix_asym = torch.randint(-50, 50, [cls.N, cls.N])
cls.v_vector = torch.randint(-50, 50, [cls.N])
cls.q_matrix = torch.triu(cls.q_matrix) + torch.triu(cls.q_matrix, diagonal=1).T
```

Reply:
Thanks for the comment. Done.